### DISCUSSION TITLE
Help interpreting MCPcounter output

### DISCUSSION THREAD

**Discussion by masaver** [omnideconv/immunedeconv#69] [General]
Hi All, 

I tested the MCPcounter implementation in immunedeconv, using the following command:
`res_MCPcounter = deconvolute( expData , "mcp_counter" , feature_types = "HUGO_symbols" )` .
And you can see the results in the following plot:

![image](https://user-images.githubusercontent.com/25987663/115399048-157e1080-a1e8-11eb-9775-ec8b29be0a9d.png)

I have two questions regarding the output scores:

1. As you can see, there are 11 groups/cell-types, while in the original publication have only 10. You can see that there are two groups, **Macrophage/Monocyte & Monocyte** , that seem to have the same values. Are they in fact the same or different from one another ?
2. One of the groups is the **cytotoxicity score**. Is this the same as the **cytotoxic lymphpocites** form the original MCPcounter publication.

Best,

-Mathias

**Comment by grst** [comment]
Hi Mathias, 

> 1. As you can see, there are 11 groups/cell-types, while in the original publication have only 10. You can see that there are two groups, Macrophage/Monocyte & Monocyte , that seem to have the same values. Are they in fact the same or different from one another ?

yes, they are exactely the same. See also https://github.com/icbi-lab/immunedeconv/issues/52. In brief, the duplicate mapping is a relict from the benchmarking project, and I'll fix it in a future version. 

> 2. One of the groups is the cytotoxicity score. Is this the same as the cytotoxic lymphpocites form the original MCPcounter publication.

Correct! To check how cell-types have been mapped, have a look at the `mapping` sheet of this excel file: https://github.com/icbi-lab/immunedeconv/blob/master/inst/extdata/cell_type_mapping.xlsx

Hope that helps, 
Gregor

**Reply by masaver** [reply]
Thanks a lot ! That helped to answer my questions. 

-Best,

Mathias
